packages/apputils-extension/style/base.css

```diff
@@ -6,3 +6,26 @@
 
 @import './notification.css';
 @import './splash.css';
+
+.jp-ContextualShortcut-TableItem {
+  white-space: pre-wrap;
+  margin-left: auto;
+  margin-right: auto;
+  background-color: white;
+  vertical-align: 'top';
+  padding-left: '2px';
+  padding-right: '2px';
+  padding-top: '1px';
+  padding-bottom: '1px';
+  color: '#888888';
+  border-style: 'solid';
+  border-width: '1px';
+  font-size: '11.7px';
+  font-family: monospace;
+}
+
+.jp-ContextualShortcut-TableRow {
+  vertical-align: 'top';
+  font-size: '13px';
+  font-family: Arial, Helvetica, sans-serif;
+}
```

Comment:
Could you move those in a specific file like `shortcuts.css` to better structure the code? Then here you can add `@import './shortcuts.css'`.